solutions/observability/logs/categorize-log-entries.md

```diff
@@ -46,7 +45,7 @@ The category row contains the following information:
 * datasets: the name of the datasets where the categories are present.
 * maximum anomaly score: the highest anomaly score in the category.
 
-To view a log message under a particular category, click the arrow at the end of the row. To further examine a message, it can be viewed in the corresponding log event on the **Stream** page or displayed in its context.
+To view a log message under a particular category, click the arrow at the end of the row. To further examine a message, it can be viewed in **Discover** or displayed in its context.
```

Comment:
```suggestion
To view a log message for a particular category, click the arrow at the end of the row. To further examine it, click **View in Discover** or **View in context**
```
Florent recently taught me we should try to avoid directional language for a11y.